docs/guides/dynamo_run.md

```diff
@@ -111,6 +112,43 @@ The `llama3B_pool` name is purely symbolic, pick anything as long as it matches
 
 Run `dynamo-run --help` for more options.
 
+### KV-aware routing
+
+**Setup**
+
+Only patched vllm currently supports KV-aware routing. Key setup steps:
+
+1. `etcd` and `nats` (see earlier) must be running and accessible from all nodes.
+1. Create a virtualenv: `uv venv kvtest`, source it's `activate`
+1. Install vllm 0.8.4: `uv pip install vllm==0.8.4`
+1. Patch it. `cd kvtest/lib/python3.12/site-packages`. `patch -p1 < $REPO_ROOT/container/deps/vllm/vllm_v0.8.4-dynamo-kv-disagg-patch.patch`
```

Comment:
could we use the package ai-dynamo-vllm pip package?

Reply:
Certainly. I will update the guide.

Reply:
Done